features/facebookImport/src/context/importer-context.jsx

```diff
@@ -125,12 +134,24 @@ export const ImporterProvider = ({ children }) => {
         return storage.removeFile(fileID);
     };
 
+    const handleSelectFile = async () => {
+        const { polyNav } = pod;
+        setFiles(null); // To show the loading overlay
```

Comment:
I'd argue it's within the scope, since this part was copied, so it's a good chance for some micro refactoring :D I guess we could either get rid of the comment, or write a little `showLoading` function or something to that effect that just sets files to null. I do think we can just remove the comment though.

Reply:
Maybe like this?